README.md

```diff
@@ -82,6 +82,28 @@ to the top header row, then save the spreadsheet.
     (save-workbook! "spreadsheet.xlsx" wb)))
 ```
 
+### Example: Create a workbook with multiple sheets
+This example creates a spreadsheet with multiple sheets. Instead of just passing a
+sheet name and a dataset, we pass a vector of the form [SheetName1, SheetData1,
+SheetName2, SheetData2, ...].
+
+```clj
+(use 'dk.ative.docjure.spreadsheet)
+
+;; Create a spreadsheet and save it
+(let [wb (create-workbook ["Squares"
```

Comment:
Personally, I'm more fond of 
`{:name "Squares
  :data [..]
}`
since I find tuples harder to read than maps